app.rb

```diff
@@ -1,9 +1,7 @@
-# app.rb
-
 require_relative 'classroom'
 require_relative 'person'
 require_relative 'students'
-require_relative 'teachers'
+require_relative 'teachers' # Updated: Added 'teacher' file
 require_relative 'books'
 require_relative 'rental'
```

Comment:
- [ ] **Optional:** Creating a rental I discover, I am ultimately entering all the records afresh, but this aught not be the case, I just need to select the persons and books in the system to place a rental. Kindly fix this going forward.

Reply:
Thank you for the observations